scripts/base/protocols/ssl/main.zeek

```diff
@@ -514,5 +518,9 @@ event analyzer_violation_info(atype: AllAnalyzers::Tag, info: AnalyzerViolationI
 	{
 	if ( atype == Analyzer::ANALYZER_SSL || atype == Analyzer::ANALYZER_DTLS )
 		if ( info$c?$ssl )
+			{
+			# analyzer errored out; prevent us from trying to remove it later
+			delete info$c$ssl$analyzer_id;
 			finish(info$c, T);
```

Comment:
Nit: That `T` actually is a indication whether we want to call `disable_analyzer()`, but that bites with above deletion of `analyzer_id`, so would be less confusing to pass `F` here.